internal/cmd/cmd.go

```diff
@@ -120,7 +120,7 @@ func CommandContext(ctx context.Context, host cow.ProcessHost, name string, arg
 	return cmd
 }
 
-func copyAndLog(w io.Writer, r io.Reader, log *logrus.Entry, name string) (int64, error) {
+func relayIO(w io.Writer, r io.Reader, log *logrus.Entry, name string) (int64, error) {
```

Comment:
A comment detailing what this function does (reads from r to w until completion or eof) and what will be logged once the relay completes might be helpful